benchmarks/smallmatvec/manual_versions/openmp/problemsize.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
```

Comment:
Please add a comment that briefly describes what this script does.

Reply:
Done